### PR TITLE
Revert the ESP maximum size back to 600MiB

### PR DESCRIPTION
**Pull request by hughsie**
The original issue was that 200MiB was too small, and we only increased the maximum for people experimenting with UKIs. Seeing as Anaconda chooses the maximum size in more cases than we'd like just drop the maximum size back down to 600MiB which is easily big enough for firmware updates.

This should fix several regressions like:

 * https://bugzilla.redhat.com/show_bug.cgi?id=2212121
 * https://bugzilla.redhat.com/show_bug.cgi?id=2214342

**Thank you** for your contribution!

Please check that your PR follows these rules:

* [ ] [**Code conventions**](https://anaconda-installer.readthedocs.io/en/latest/contributing.html#code-conventions). tl;dr: Follow PEP8, wrap at 100 chars, and provide docstrings.

* [ ] [**Commit message conventions**](https://anaconda-installer.readthedocs.io/en/latest/commit-log.html). tl;dr: Heading, empty line, longer explanations, all wrapped manually. If in doubt, write a longer commit message with more details.

* [ ] **Tests** pass and cover your changes. You can [run tests locally manually](https://anaconda-installer.readthedocs.io/en/latest/testing.html), or have them run as part of the PR. A team member will have to enable tests manually after inspecting the PR.
  *If you don't know how, ask us for help!*

* [ ] [**Release notes**](https://anaconda-installer.readthedocs.io/en/latest/contributing.html#release-notes) and **docs** if the PR adds something major or changes existing behavior.
  *If you don't know how, ask us for help!*

* [ ] **RHEL** rules: If your PR is for a `rhel-*` branch, pay special attention to commit messages. Make sure **all** commit messages include a line linking the commit to a bug, such as `Resolves: rhbz#123456`. For more information, see [the complete rules](https://anaconda-installer.readthedocs.io/en/latest/commit-log.html).
  *If you don't know how, ask us for help!*

Don't forget - *if you don't know how, ask us for help!*

And now that you read this all, you can delete it and type your own description of your changes :-)

### CODE REVIEW

**Comment by hughsie** [comment]
@AdamWill

**Comment by hughsie** [comment]
> Release note check

This failed, but I think it's okay as the original PR to change the default included a release note.

**Review by VladimirSlavik — APPROVED** [review]
You might want to drop also `docs/release-notes/bigger-esp.rst`. Otherwise LGTM. Thank you for taking care of this!

**Comment by VladimirSlavik** [comment]
For future records - this reverts #4711 and c253390b23475970f1d696b4fb9d719a0c316559 (sans the release note)

**Comment by AdamWill** [comment]
> You might want to drop also `docs/release-notes/bigger-esp.rst`. Otherwise LGTM. Thank you for taking care of this!

well, it should probably be amended? The ESP will still be *bigger* than before we started changing things here, just not so much bigger.

**Comment by VladimirSlavik** [comment]
Oh. I missed that completely, thank you. In that case it's fine.

**Comment by AdamWill** [comment]
I would say it's not *completely* accurate as written. I would change Description to:

"The minimum size of the EFI System Partition (ESP) created by Anaconda has changed from 200 MiB to 500 MiB. The maximum size, which is used in most cases, remains at 600 MiB."

**Comment by hughsie** [comment]
Want me to update it to that? Im fine if someone wants to rewrite it on merge.

**Comment by hughsie** [comment]
> Want me to update it to that

Narrator: _He did that._

**Comment by VladimirSlavik** [comment]
Interpretation of the storage kickstart tests results:

- Known failures:
  - autopart-luks-1 = gh774
  - raid-1 = gh777
  - storage-multipath-autopart = rhbz1853668
- Failures without explanation:
  - lvm-luks-1
  - lvm-luks-2
  - lvm-luks-3
  - lvm-luks-4
  - part-luks-1
  - part-luks-2
  - part-luks-3
  - part-luks-4
  - raid-luks-1
  - raid-luks-2 
  - raid-luks-3
  - raid-luks-4

... I think I see a theme there. Onwards to logs...

**Comment by VladimirSlavik** [comment]
tl;dr I hope the commonality for LVM and part tests is the (explicit) storage layout in these tests, which is no longer valid. For the raid-luks group, it's more mysterious but the installation appears to succeed despite

----

Case by case:

- `lvm-luks-*` fail because "Unable to allocate requested partition scheme." and manual intervention is needed
- `part-luks-*` ditto
- `raid-luks-{1,2,4}` appear to install successfully, but validation of the test fails because `cat: /var/tmp/kstest-****/RESULT: No such file or directory`; the %post script seems to run ok with exit 0 though...
- `raid-luks-3` dies because of "gnome-kiosk exited with status 1" which async kills the installer; this is highly likely to be unrelated to the PR changes and a fluke, hopefully

I *think* this indicates that mostly the tests need rewriting for the new storage layout, but I'll have to verify this manually.

Next: Run the same kickstart scenario manually, investigate the VM.

@hughsie @Adamwill Is this intended for beta already, or later? From the [FE list](https://qa.fedoraproject.org/blockerbugs/milestone/39/beta/buglist) I gather this is meant to go post-Beta, right?

**Comment by rvykydal** [comment]
>     * `raid-luks-{1,2,4}` appear to install successfully, but validation of the test fails because `cat: /var/tmp/kstest-****/RESULT: No such file or directory`; the %post script seems to run ok with exit 0 though...
> 
>     * `raid-luks-3` dies because of "gnome-kiosk exited with status 1" which async kills the installer; this is highly likely to be unrelated to the PR changes and a fluke, hopefully

I agree, probably https://github.com/rhinstaller/kickstart-tests/issues/997

**Comment by VladimirSlavik** [comment]
Well, duh. I missed that all of the failed tests are manual... they fail without the change as well.
```
$kickstart-tests> grep TESTTYPE {part,lvm,raid}-luks-* | grep manual
part-luks-1.sh:TESTTYPE="manual storage partition luks"
part-luks-2.sh:TESTTYPE="manual storage partition luks"
part-luks-3.sh:TESTTYPE="manual storage partition luks"
part-luks-4.sh:TESTTYPE="manual storage partition luks"
lvm-luks-1.sh:TESTTYPE="manual storage lvm luks"
lvm-luks-2.sh:TESTTYPE="manual storage lvm luks"
lvm-luks-3.sh:TESTTYPE="manual storage lvm luks"
lvm-luks-4.sh:TESTTYPE="manual storage lvm luks"
raid-luks-1.sh:TESTTYPE="manual storage raid luks"
raid-luks-2.sh:TESTTYPE="manual storage raid luks"
raid-luks-3.sh:TESTTYPE="manual storage raid luks"
raid-luks-4.sh:TESTTYPE="manual storage raid luks"
```

(I think the bit that tripped me is that we usually add the disable-tags at the end of the list, but manual is at front.)
